### PR TITLE
Search parameters

### PR DESCRIPTION
**Pull request by lee-richardson-nhs**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/immunisation-fhir-api.yaml [hunk 1]
```diff
@@ -501,24 +521,48 @@ components:
...
       in: query
-      name: -diseaseType
+      name: -immunization.target
       description: |
         The vaccine preventable disease the dose is being administered against.
         Expressed as `<value>` where`<value>` must be a valid Snomed Code.
       required: true
       schema:
         type: string
         example: "39114911000001105"
```
Comment (nathanwall) [inline specification/immunisation-fhir-api.yaml:541]: Does this example need to be COVID19, FLU, HPV or MMR?

Comment (lee-richardson-nhs) [reply specification/immunisation-fhir-api.yaml]: Yes, and I've replaced the description with a modified one from Imms History.

**Review by nathanwall — CHANGES_REQUESTED** [review]
Looks great. Should be great to merge once the rest of your tests are done.

File: backend/tests/test_immunization_post_validator_model.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
     load_json_data,
 )
 from tests.utils.mandation_test_utils import MandationTests
+from src.models.utils.post_validation_utils import MandatoryError
```
Comment (nathanwall) [inline backend/tests/test_immunization_post_validator_model.py:16]: Is this import needed?

Comment (lee-richardson-nhs) [reply backend/tests/test_immunization_post_validator_model.py]: It is not. No idea where that came from.

**Review by nathanwall — APPROVED** [review]

**Pull request merged**